app/models/miq_provision_workflow.rb

```diff
@@ -3,17 +3,23 @@ def self.base_model
     MiqProvisionWorkflow
   end
 
+  # Find the appropriate workflow class for the given 'platform' string.
+  #
+  # @example openstack
+  #   "ManageIQ::Providers::Openstack::CloudManager::ProvisionWorkflow"
+  #
+  # @param platform [String]
+  #   A string of the one of the ManageIQ providers. The case of this
+  #   string is ignored.
+  #
+  # @return [String] A scoped provider constant name.
```

Comment:
This returns a class, not a string

Reply:
@djberg96 I think we can merge once this is corrected

Reply:
@bdunne Updated.